websites/I/Instagram/presence.ts

```diff
@@ -1,61 +1,41 @@
-var presence = new Presence({
+const presence = new Presence({
   clientId: "547436289960574977"
 });
 
 presence.on("UpdateData", async () => {
-  if (document.location.pathname == "/") {
-    const homepagePresence: PresenceData = {
-      details: "Viewing the homepage",
-      largeImageKey: "logo"
-    };
-    presence.setActivity(homepagePresence);
+	const presenceData: PresenceData = {
+		largeImageKey: "logo"
+	};
+
+  if (document.location.pathname === "/") {
+      presenceData.details = "Viewing the homepage";
   } else if (document.location.pathname.startsWith("/stories")) {
-    const presenceData: PresenceData = {
-      details: "Viewing a story",
-      state: document.location.pathname.split("/")[2],
-      largeImageKey: "logo"
-    };
-    presence.setActivity(presenceData);
+      presenceData.details = "Viewing a story";
+      presenceData.state = document.location.pathname.split("/")[2];
   } else if (document.location.pathname.startsWith("/accounts")) {
-    const presenceData: PresenceData = {
-      details: "Settings",
-      state: "Changing their Settings",
-      largeImageKey: "logo"
-    };
-    presence.setActivity(presenceData);
+      presenceData.details = "Settings";
+      presenceData.state = "Changing their Settings";
   } else if (document.location.pathname.startsWith("/p")) {
-    const presenceData: PresenceData = {
-      details: "Viewing a post",
-      largeImageKey: "logo"
-    };
-    presence.setActivity(presenceData);
+      presenceData.details = "Viewing a post";
   } else if (document.location.pathname.startsWith("/explore")) {
-    const presenceData: PresenceData = {
-      details: "Exploring...",
-      largeImageKey: "logo"
-    };
-    presence.setActivity(presenceData);
+      presenceData.details = "Exploring...";
   } else if (document.location.pathname.startsWith("/nametag")) {
-    const presenceData: PresenceData = {
-      largeImageKey: "logo"
-    };
-    presence.setActivity(presenceData);
+      presenceData.details = "Viewing nametag";
   } else if (
     document.location.pathname.startsWith("/direct/inbox") ||
     document.location.pathname.startsWith("/direct/t")
   ) {
-    const presenceData: PresenceData = {
-      details: "Direct Messages",
-      largeImageKey: "logo"
-    };
-    presence.setActivity(presenceData);
+      presenceData.details = "Direct Messages";
   } else {
     // TODO: Check if the page is really a profile
-    const presenceData: PresenceData = {
-      details: "Viewing a profile",
-      state: document.location.pathname.split("/")[1],
-      largeImageKey: "logo"
-    };
+      presenceData.details = "Viewing a profile";
+      presenceData.state = document.location.pathname.split("/")[1];
+    }
+	
+	if (presenceData.details === null) {
```

Comment:
from what I see - details is always a thing so maybe remove this if and leave only function from else :/